docs/05-entity-component-system.mdx

```diff
@@ -3,30 +3,32 @@ title: ECS
 path: /docs/entity-component-system
 ---
 
-Excalibur has a built in Entity Component System (ECS for short), which is a popular software technique in the video game industry
-for managing behavior in a game in a composable and reusable way. Many ECS implementations also focus on memory cache optimization
-techniques, that is not a goal of the current Excalibur implemenation.
+// TODO does a first-time user need to internalize our ECS structure?
```

Comment:
Probably not, but I think it is useful to explain the reasoning behind it?

Reply:
Oh absolutely, but I think maybe we can flag certain sections of the docs as "optional" reading for people who are just getting started, and then other sections as "recommended" reading.